extensions/grpc/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: "gRPC"
+metadata:
+  keywords:
+    - "gRPC"
+  categories:
+    - "grpc"
```

Comment:
Shouldn't it be in the serialization category instead? I don't think we want a category for one extension?

Reply:
Protobuf would be serialization. Here we have a server and a client mechanism, not just a format. 

I looked at other categories and I would propose: 

* reactive
* web (netty, for example, is in web, as well as the rest client)
* serialization (because of protobuf)

Reply:
Yes, these categories look good.